workspaces/resource-optimization/plugins/resource-optimization/src/components/OptimizationsBreakdownChart/utils/chart-data-format.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  LongTermRecommendationBoxPlots,
+  MediumTermRecommendationBoxPlots,
+  RecommendationBoxPlotsRecommendations,
+  ShortTermRecommendationBoxPlots,
+} from '@backstage-community/plugin-resource-optimization-common';
+import {
+  Interval,
+  OptimizationType,
+  RecommendationType,
+  ResourceType,
+  UsageType,
+} from '../types/chart';
+import { format } from 'date-fns';
+
+export const getRecommendationTerm = (
+  interval: Interval,
+  recommendations?: RecommendationBoxPlotsRecommendations,
+):
+  | ShortTermRecommendationBoxPlots
+  | MediumTermRecommendationBoxPlots
+  | LongTermRecommendationBoxPlots => {
+  let result;
+  switch (interval) {
```

Comment:
I see linter warning on this block because there is no default case. Please add an exception (`// eslint-disable-next-line default-case`) to denote the missing default case is missing intentionally